kano_settings/parental_config.py

```diff
@@ -153,8 +156,19 @@ def __init__(self, size_x=250, size_y=25):
         EditableList.__init__(self, size_x=size_x, size_y=size_y)
 
     @staticmethod
-    def _sanitise_site(site):
-        return site.replace(' ', '').lstrip('www.')
+    def _sanitise_site(url):
+        parse = urlparse(url)
+
+        # enforcing some standards (e.g. http://)
+        if parse.scheme == '' or parse.netloc == '':
```

Comment:
I don't think that we should be making this harder for the user than it already is. @convolu @convolu what do you think?